web/.c8rc.json

```diff
@@ -7,7 +7,17 @@
   "reporter": ["text", "text-summary"],
   "reports-dir": "build/coverage",
   "src": [
-    "**/src/",
+    "src/**",
     "../common/**/src"
+  ],
+  "include": [
+    "src/**"
+  ],
+  "exclude": [
+    "src/test/**",
+    "**/src/tests/**",
+    "**/build/**",
```

Comment:
Do we still have `/test/` folders as well as `/tests/`?

Reply:
Whoops, put a comment related to this in the wrong place on this PR:  https://github.com/keymanapp/keyman/pull/13196/files#r1951926796

One of the child projects has a /tests/ folder deeper in the hierarchy.